tasks/test.js

```diff
@@ -64,6 +64,16 @@ module.exports = function (grunt) {
     'stop:testUIServer'
   ]);
 
+  grunt.registerTask('test:uiRelease', [
+    'checkPlugins',
+    'esvm:ui',
+    'run:testUIReleaseServer',
+    'clean:screenshots',
+    'functionalTestRunner',
+    'esvm_shutdown:ui',
+    'stop:testUIReleaseServer'
+  ]);
+
```

Comment:
Also copypasta. Hm. I suppose one could pass in an array that's reused except for certain items, but I don't think it looks so great in this case. Would be super unclear indirection.